addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -164,6 +171,12 @@ public void connect() {
         } catch (URISyntaxException e) {
             logger.error("Error creating Socket.IO: {}", e.getMessage());
         }
+        try {
```

Comment:
In my opinion you should collapse the two try-catch blocks, there goal is to separate the error handling from the flow of code, you could also consider to extract the try-catch to a seperate method maybe that is even better.